### PR TITLE
Add descriptions

### PR DESCRIPTION
**Pull request by thepassle**
Add descriptions to items as per awesome list instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,27 +33,29 @@ lit-html — an efficient, expressive, extensible HTML templating library for Ja
...
+- [Material Web Components](https://github.com/material-components/material-components-web-components), material design implemented as Web Components.
+- [Wired Elements](https://github.com/wiredjs/wired-elements), collection of elements that appear hand drawn.
 
 ## Starter Templates
 
-- [PWA Starter Kit](https://github.com/Polymer/pwa-starter-kit)
-- [Create-lit-app](https://github.com/thepassle/create-lit-app)
+- [PWA Starter Kit](https://github.com/Polymer/pwa-starter-kit), a starter template by the Polymer team.
+- [Create-lit-app](https://github.com/thepassle/create-lit-app), create lit-html apps with no build configuration.
+
```
Comment (web-padawan) [inline README.md]: nit: these extra newlines here seem unnecessary, so please remove

File: README.md [hunk 2]
```diff
@@ -33,27 +33,29 @@ lit-html — an efficient, expressive, extensible HTML templating library for Ja
...
-- [@littleq/element-lite](https://github.com/tjmonsi/element-lite)
-- [@corpuscule/element](https://github.com/corpusculejs/corpuscule)
-- [@dabolus/localized-lit-element](https://github.com/Dabolus/localized-lit-element)
-- [lit-apollo](https://github.com/bennypowers/lit-apollo)
+- [@polymer/lit-element](https://github.com/Polymer/lit-element), an ultra-light custom element base class with a simple but expressive API.
+- [gluonjs](https://github.com/ruphin/gluonjs), a lightweight library for building Web Components and applications.
+- [lit-element](https://github.com/DiiLord/lit-element), implements lit-html via a LitElement class.
+- [lit-html-element](https://github.com/kenchris/lit-element), a base class for creating Web Components using lit-html.
+- [fit-html](https://github.com/Festify/fit-html), fit-html is a combination of lit-html, Web Components and Redux.
+- [lit-html-brackets](https://github.com/bgotink/lit-html-brackets), lit-html extension that uses a bracket syntax similar to angular's template syntax.
```
Comment (web-padawan) [inline README.md]: please capitalise "Angular"

File: README.md [hunk 3]
```diff
@@ -33,27 +33,29 @@ lit-html — an efficient, expressive, extensible HTML templating library for Ja
...
+- [fabys-lit-element](https://github.com/fabysdev/fabys-lit-element), a base class for creating ES6 Web Components using TypeScript and lit-html.
```
Comment (web-padawan) [inline README.md]: please remove "ES6", although repo description contains it. This is not necessary, and other descriptions do not contain it so better to be aligned with these

**Comment by thepassle** [comment]
Fixed :-)

**Comment by web-padawan** [comment]
LGTM 👍

**Pull request merged**